scripting/get5/pausing.sp

```diff
@@ -1,8 +1,17 @@
 public bool Pauseable() {
-  return g_GameState >= Get5State_KnifeRound && g_PausingEnabledCvar.BoolValue;
+  return g_GameState == Get5State_Live && g_PausingEnabledCvar.BoolValue;
 }
 
-public bool PauseGame(Get5Team team, Get5PauseType type, int pausesLeft) {
+stock void PauseGame(Get5Team team, Get5PauseType type) {
```

Comment:
Doesn't really matter, but it would make more sense to remain 'public'.

Reply:
Was changed because it had default values which are unsupported when using public. It doesn't anymore though, after I removed in-game-state pauses, so I can change it back. I still don't understand the difference entirely. It's not a callback and it's not being used outside the mod.

Only obvious advantage I see to public here is that we get the "not used" warning from the compiler if we someway don't use it.

I'll change it back.